modules/ROOT/pages/creating-an-odatav2-api-with-apikit.adoc

```diff
@@ -27,7 +25,7 @@ The following example shows you how to implement an OData-enabled REST API:
 . Copy the `odata.raml` file you downloaded in the prerequisites section to the `/src/main/resources/api` directory in your project.
 . In the project explorer, right-click `odata.raml` and select *Mule* > *Generate OData API from RAML Types*.
 +
-image::generate-odata.png["Window highlights the full path that was previously mentioned to get to the generate OData API from RAML types option."]
+image::generate-odata.png["Window highlights the full path that was mentioned before to get to the generate OData API from RAML types option."]
```

Comment:
Not sure if the "full path mentioned before" part is clear. Could you rewrite it to something like: 
"A window highlighting the full path to the generate OData API from RAML types option." ? (Assuming this is accurate)

Reply:
I like it and it's right! Thanks!